src/components/PaymentRequestActionMenu/PaymentRequestActionMenu.tsx

```diff
@@ -0,0 +1,82 @@
+import { cva } from 'class-variance-authority';
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu';
+import menuIcon from '../../assets/images/nf_menu.svg';
+import copyIcon from '../../assets/images/nf_copy.svg';
+import linkIcon from '../../assets/images/nf_link.svg';
+import trashIcon from '../../assets/images/nf_trash.svg';
+
+const actionItem = cva(
+  'group text-xs leading-none rounded-1 flex items-center h-6 py-4 pr-4 relative select-none outline-none cursor-pointer data-[highlighted]:text-greyText',
+  {
+    variants: {
+      intent: {
+        neutral: [],
+        negative: ['text-red-700'],
+      },
+    },
+    defaultVariants: {
+      intent: 'neutral',
+    },
+  },
+);
```

Comment:
We could remove this as no intents are being used.

Maybe keep the default classes in a const variable, and reuse it.

Something like:
```jsx
// ..
const itemClassnames = "group text-xs leading-none rounded-1 flex items-center h-6 py-4 pr-4 relative select-none outline-none cursor-pointer data-[highlighted]:text-greyText"
// ...
```

And then use it as:
```jsx
{/* ... */}

<DropdownMenu.Item className={itemClassnames} onClick={onDuplicate}>

{/* ... */}

<DropdownMenu.Item className={itemClassnames} onClick={onCopyLink}>

{/* ... */}

<DropdownMenu.Item className={itemClassnames} onClick={onDelete}>

{/* ... */}

```